cicecore/cicedyn/general/ice_init.F90

```diff
@@ -1414,6 +1419,12 @@ subroutine input_data
             write(nu_diag,*) subname//' WARNING:   For consistency, set tfrz_option = mushy'
          endif
       endif
+      if (ktherm == 1 .and. trim(saltflux_option) /= 'constant') then
```

Comment:
should this be an error or just a warning that nobody will see?

Reply:
I'm basically going with the example of tfrz_option here. It is not advisable to use ktherm = 1 and saltflux_option /= 'constant', but maybe people could do it if they wanted.

Reply:
fine by me, was just asking.